lib/kv-redis.js

```diff
@@ -42,10 +63,30 @@ function RedisKeyValueConnector(settings, dataSource) {
 
 util.inherits(RedisKeyValueConnector, Connector);
 
+/**
+ * Returns "PONG". Used to verify connectivity to the data source.
+ *
+ * @callback {Function} callback
+ * @param {Error} err Error object.
+ * @param {String} message The "PONG" message string.
+ *
+ * @header RedisKeyValueConnector.ping(cb)
```

Comment:
We do it everyone where else, see https://github.com/strongloop/loopback/blob/master/lib/persisted-model.js#L485 for example. That said, I'm on your side to add it back in everywhere as I never understood why it was left out in the first place. Time to make a new convention, which is to do it properly IMO.